utils/docker-ce/Makefile

```diff
@@ -30,8 +30,9 @@ define Package/docker-ce
   CATEGORY:=Utilities
   TITLE:=Docker Community Edition
   URL:=https://www.docker.com/
-  DEPENDS:=$(GO_ARCH_DEPENDS) @TARGET_x86_64 +btrfs-progs +libdevmapper +containerd +cgroupfs-mount +ca-certificates +iptables-mod-extra +DOCKER_SECCOMP:libseccomp \
-           +kmod-ikconfig +kmod-veth +kmod-br-netfilter +kmod-nf-ipvs
+  DEPENDS:=$(GO_ARCH_DEPENDS) @TARGET_x86_64 +containerd +tini +libnetwork +cgroupfs-mount +btrfs-progs +libdevmapper +ca-certificates \
+           +iptables-mod-extra +DOCKER_SECCOMP:libseccomp +kmod-ikconfig +kmod-veth +kmod-br-netfilter +kmod-nf-ipvs
+  EXTRA_DEPENDS:=containerd (=1.2.6-1) runc (=1.0.0-rc8-1) tini (=0.18.0-1) libnetwork (=0.8.0-dev.2-1)
```

Comment:
Better leave this out.  This is too specific that I am afraid it could hurt `opkg upgrade` and add unnecessary maintenance burden.

Reply:
I was concerned about the compatibility of containerd with docker if I were to update containerd to 1.2.7. I'll just leave updating containerd until I update docker to 19.03.0 instead.

Reply:
If we were to do this, I would suggest maintaining such version constraints with Makefile construct.  It helps to ensure compatibility among components in specific OpenWrt release.  Less rigid but helpful.

Reply:
Not quite sure what you meant about the Makefile constructs, but the only other ways to lock the version that I can think of, is having two separate packages. which seems like it would be a bit of overhead.

Reply:
Not sure about how separate packages may work here ;)

I meant something seafile-server uses right now:

https://github.com/openwrt/packages/blob/c8734591ca2d2c9e69a69591ccd1365795826689/net/seafile-server/Makefile#L28-L35

Reply:
This is not a hard requirement though

Reply:
Thanks for the advice, added something similar.